client/blocks/applause/edit.js

```diff
@@ -36,7 +36,7 @@ const EditApplauseBlock = ( props ) => {
 		setAttributes
 	);
 
-	const accountInfo = useAccountInfo();
+	const accountInfo = get( useAccountInfo(), 'data', {} );
```

Comment:
Nit: ```const { data: accountInfo } = useAccountInfo();```

Though it's not an issue with a simple `get()` I think it's best to avoid adding any logic wrappers around hook as it can cause rendering issues.